antbs/repo_monitor.py

```diff
@@ -36,7 +36,6 @@
 import logging
 
 import requests
-from github3 import login
 from gitlab import Gitlab
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/antbs/repo_monitor.py
+++ b/antbs/repo_monitor.py
@@ -36,7 +36,6 @@
 import logging
 
 import requests
-from gitlab import Gitlab
 
 import iso
 import webhook
@@ -44,6 +43,7 @@
 from database.base_objects import db
 from database.server_status import status
 from github3 import login
+from gitlab import Gitlab
 from utils.logging_config import logger
 
 GITLAB_TOKEN = status.gitlab_token
```